Dockerfile

```diff
@@ -374,7 +374,8 @@ COPY --from=zmq-plugin --chown=tenzir:tenzir /plugin/zmq /
 
 FROM tenzir-de AS tenzir-node-de
 
-ENTRYPOINT ["tenzir-node"]
+USER root:root
+ENTRYPOINT ["capsh", "--caps=cap_net_bind_service,cap_setuid,cap_setgid=+eip", "--user=tenzir", "--addamb=cap_net_bind_service", "--", "-c", "/opt/tenzir/bin/tenzir-node"]
```

Comment:
Note that this will not work in a few Docker Compose files we have out in the wild which override the entrypoint.

Reply:
Take a look at these services from our Docker Compose. They use different entrypoints, and intentionally specify the same image as to avoid having to download the image twice.

https://github.com/tenzir/tenzir/blob/2f8b4af71297f55d997b63d70ac7f6806a54208f/docker-compose.yaml#L4-L21

https://github.com/tenzir/tenzir/blob/2f8b4af71297f55d997b63d70ac7f6806a54208f/docker-compose.yaml#L43-L55

This would break with the changes from this PR.